setup.py

```diff
@@ -16,6 +16,7 @@
     "python-dateutil", "prettytable", "h5py",
     "qasync", "pyqtgraph", "pygit2",
     "llvmlite_artiq", "pythonparser", "python-Levenshtein",
+    "jsonschema",
```

Comment:
This may cause user installations with conda to fail. Anyway the gateware building dependencies are optional and should not be included there.